src/DETHRACE/constants.h

```diff
@@ -491,7 +491,7 @@ enum {
 
 #define TIME_CONV_THING 0.0005f
 
-#define OPPONENT_COUNT 0
+#define OPPONENT_COUNT 5
```

Comment:
Haha, perhaps a `--opponent-count` argument option could be introduced?

Reply:
haha yeah, I've thought the same a few times and havent got around to it :)